sonar-db/src/main/java/org/sonar/db/purge/PurgeMapper.java

```diff
@@ -33,8 +32,6 @@
    */
   List<IdUuidPair> selectComponentsByProjectUuid(String projectUuid);
 
-  void selectComponentUuidsToDisable(@Param("rootUuid") String rootUuid, ResultHandler resultHandler);
-
   void deleteAnalyses(@Param("analysisUuids") List<String> analysisUuids);
 
   void deleteAnalysisDuplications(@Param("analysisUuids") List<String> analysisUuids);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUndocumentedApi)